docs/root/configuration/http/http_filters/on_demand_updates_filter.rst

```diff
@@ -1,15 +1,19 @@
 .. _config_http_filters_on_demand:
 
-On-demand VHDS Updates
-======================
+On-demand VHDS ans S/RDS Updates
+================================
 
-The on-demand VHDS filter is used to request a :ref:`virtual host <envoy_v3_api_msg_config.route.v3.VirtualHost>`
+The on-demand update filter can be used to request a :ref:`virtual host <envoy_v3_api_msg_config.route.v3.VirtualHost>`
```

Comment:
Maybe add an introductory sentence explaining the filter serves two possible purposes.

Reply:
Will do.